test/test_intensify/test_successive_halving.py

```diff
@@ -971,3 +972,148 @@ def test_incumbent_selection_designs(self):
         inc = intensifier._compare_configs(incumbent=self.config4, challenger=self.config3,
                                            run_history=self.rh, log_traj=False)
         self.assertEqual(self.config3, inc)
+
+    def test_count_running_instances_for_challenger(self):
+        """
+        Makes sure we return the proper number of running configs
+        """
+        def target(x):
+            return 1
+        taf = ExecuteTAFuncDict(ta=target, stats=self.stats, run_obj='quality')
+
+        taf.runhistory = self.rh
+        # select best on any budget
+        intensifier = SuccessiveHalving(
+            stats=self.stats, traj_logger=None,
+            rng=np.random.RandomState(12345), run_obj_time=False,
+            instances=list(range(20)), initial_budget=3, max_budget=10, eta=2)
+
+        # Get and process the first instance pair of config 1
```

Comment:
Could you please add a check for an empty run history?

Reply:
I added a new check, but maybe I misunderstood this. Please let me know if the new check is what you were looking for